meson_options.txt

```diff
@@ -14,6 +14,8 @@ option('docs', type: 'feature', value: 'auto',
     description: 'Build documentation')
 option('omf-byte-order', type: 'combo', choices: ['native', 'big', 'little'],
     value: 'native', description: 'On-media-format byte order')
+option('kvs-key-cpe', type: 'combo', choices: ['speed', 'size'],
+    value: 'size', description: 'KVS Key Common-Prefix-Elimination')
```

Comment:
Can you expand a bit in the description? `Optimization for key common prefix elimination`. Since I assume this means optimizing for speed or size.

Reply:
Right...